api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -12,6 +12,8 @@
 public class AttachmentsConstants {
 	
 	public static enum ContentFamily {
+		EXECUTABLE,
+		SCRIPT,
```

Comment:
@jnsereko you basically treat everything as executables in this PR, there is no reason to distinguish scripts from executables.

Reply:
oohh..  great suggestion @mks-d.  Thanks